stream-compositions/services/product-composition-service/src/main/java/com/backbase/stream/compositions/product/core/service/impl/ProductIntegrationServiceImpl.java

```diff
@@ -9,6 +9,7 @@
 import com.backbase.stream.legalentity.model.ProductGroup;
 import lombok.AllArgsConstructor;
 import lombok.extern.slf4j.Slf4j;
+import org.jetbrains.annotations.NotNull;
```

Comment:
unused import

Reply:
Removed unused import as well. Thanks for highlighting.